cabot/cabotapp/models.py

```diff
@@ -141,6 +144,17 @@ class Meta:
         help_text='Gist, Hackpad or Refheap js embed with recovery instructions e.g. '
                   'https://you.hackpad.com/some_document.js'
     )
+    hipchat_instance = models.ForeignKey(
```

Comment:
Instances have Hipchat alerts on by default, so technically if we only added it to Service this would break alerts for instances. Not a big deal since we don't use instances, but in case we or anyone else ever tries to use instances might be better to just keep it in CheckGroupMixin?